<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PainterConfig</class>
 <widget class="QDialog" name="PainterConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Streamlines</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Strategy:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="strategy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Seed - Euler</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Properties:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="properties">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>351</width>
       <height>221</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="update">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Isolines</string>
    </attribute>
    <widget class="QRadioButton" name="equallySpaced">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Equally spaced:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Count:</string>
     </property>
    </widget>
    <widget class="ImprovedLineEdit" name="count">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fixedUpdate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="custom">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom:</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="values">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <column>
      <property name="text">
       <string>Isoline Value</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="updateIsolines">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="addRow">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Row</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteRow">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortIsolines">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort</string>
     </property>
    </widget>
    <widget class="QPushButton" name="randomIsolines">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Suggest</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="randomQuantity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Stream">
    <attribute name="title">
     <string>Stream Func</string>
    </attribute>
    <widget class="QComboBox" name="streamIntegrationX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>x- -&gt; x+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x+ -&gt; x-</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="streamIntegrationY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>y- -&gt; y+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y+ -&gt; y-</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Integration direction:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImprovedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>improvedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>strategy</tabstop>
  <tabstop>properties</tabstop>
  <tabstop>update</tabstop>
  <tabstop>equallySpaced</tabstop>
  <tabstop>count</tabstop>
  <tabstop>custom</tabstop>
  <tabstop>values</tabstop>
  <tabstop>randomQuantity</tabstop>
  <tabstop>randomIsolines</tabstop>
  <tabstop>sortIsolines</tabstop>
  <tabstop>addRow</tabstop>
  <tabstop>deleteRow</tabstop>
  <tabstop>updateIsolines</tabstop>
  <tabstop>streamIntegrationX</tabstop>
  <tabstop>streamIntegrationY</tabstop>
  <tabstop>fixedUpdate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
